qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -175,6 +175,86 @@ def test_mkdir_path_with_path_option(self):
         o = self.mount_a.stat('d5/d6/d7')
         log.info("mount_a output:\n{}".format(o))
 
+class TestRMdir(TestCephFSShell):
```

Comment:
`Rmdir` would be preferable since we have a class named `Mkdir`.

Reply:
Done.